### PR TITLE
Only show admin links list ifAdmin

### PR DESCRIPTION
**Pull request by paulproteus**
Testing done:

* Manually tested with a non-admin dev account: got no admin link.

* Manually tested with an admin dev account: got the admin link.

### CODE REVIEW

File: shell/packages/sandstorm-accounts-ui/login_buttons.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ var helpers = {
   isDemoUser: function () {
     return this._db.isDemoUser();
   },
+  isAdmin: function () {
+    return Meteor.user().isAdmin;
```
Comment (paulproteus) [inline shell/packages/sandstorm-accounts-ui/login_buttons.js]: Question to e.g. Drew: I notice we're access `this._db` but it seems more idiomatic to access `Meteor.user()`, so I went with what I think is more idiomatic.

Should I do things more like `isDemoUser` and access `this._db` instead?

Comment (zarvox) [inline shell/packages/sandstorm-accounts-ui/login_buttons.js]: Prefer `return this._db.isAdmin()` here.

Comment (zarvox) [inline shell/packages/sandstorm-accounts-ui/login_buttons.js]: To be more clear: in the longer future, we intend to have SandstormDb wrap all accesses to the collections, and that includes Meteor's internal collections, so we prefer to wrap that here.

It should make it easier for us to 1) mock out SandstormDb in unit tests and 2) eventually shift away from Mongo someday, as described in [The Great Frontend Refactor](https://oasis.sandstorm.io/grain/kj9fopjyT2CdqC3i7fZN2z).

Comment (paulproteus) [inline shell/packages/sandstorm-accounts-ui/login_buttons.js]: Such +1. Thanks for taking the time to explain

**Comment by paulproteus** [comment]
@zarvox I believe this is ready to merge, as soon as Jenkins says +1.

**Comment by paulproteus** [comment]
Also yay!​

**Pull request merged**